6.4_remember_remember.py

```diff
@@ -0,0 +1,32 @@
+from PIL import Image
+import os.path
+
+MY_PATH = "resources//code.png"
+
+
+def decryption(path):
+    """
```

Comment:
The code in general is not clean, I would suggest to separate code to small functions- 
https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be

Reply:
I added a function and changed the For loops to writing suitable for Python